ocaml/xapi/xapi_vtpm.ml

```diff
@@ -12,19 +12,35 @@
    GNU Lesser General Public License for more details.
  *)
 
+let assert_no_vtpm_associated ~__context vm =
+  match Db.VM.get_VTPMs ~__context ~self:vm with
+  | [] ->
+      ()
+  | vtpms ->
+      let uuids =
+        List.map (fun v -> Db.VTPM.get_uuid ~__context ~self:v) vtpms
+      in
+      raise Api_errors.(Server_error (vtpm_max_amount_reached, uuids))
```

Comment:
This should return refs, not uuids (as usual in the API). But doesn't it make more sense to return the number?

Reply:
I just thought having all the references would be more useful that just having the number, but thinking about it it the information doesn't seem to be very actuable